### ISSUE TITLE
Crash on macOS Sierra

### ISSUE THREAD

**Issue by pablopda** [scakemyer/plugin.video.quasar#564]
New version is crashing
mac sierraOS 10.12.1 
Version: "v0.9.36" Go: go1.7.3

```
23:43:27 T:123145351651328  NOTICE: [plugin.video.quasar] panic: runtime error: invalid memory address or nil pointer dereference
23:43:27 T:123145351651328  NOTICE: [plugin.video.quasar] [signal SIGSEGV: segmentation violation code=0x1 addr=0x158 pc=0x4142eb1]
23:43:27 T:123145351651328  NOTICE: [plugin.video.quasar]
23:43:27 T:123145351651328  NOTICE: [plugin.video.quasar] goroutine 274 [running]:
23:43:27 T:123145351651328  NOTICE: [plugin.video.quasar] panic(0x4a08520, 0xc42000c0b0)
23:43:27 T:123145351651328  NOTICE: [plugin.video.quasar] 	/usr/local/go/src/runtime/panic.go:500 +0x1a1
23:43:27 T:123145351651328  NOTICE: [plugin.video.quasar] github.com/scakemyer/quasar/bittorrent.(*BTPlayer).CheckAvailableSpace(0xc420214480, 0x3)
23:43:27 T:123145351651328  NOTICE: [plugin.video.quasar] 	/go/src/github.com/scakemyer/quasar/bittorrent/player.go:212 +0x131
23:43:27 T:123145351651328  NOTICE: [plugin.video.quasar] github.com/scakemyer/quasar/bittorrent.(*BTPlayer).onStateChanged(0xc420214480, 0x4fbcfe0, 0xc420452070)
23:43:27 T:123145351651328  NOTICE: [plugin.video.quasar] 	/go/src/github.com/scakemyer/quasar/bittorrent/player.go:381 +0x240
23:43:27 T:123145351651328  NOTICE: [plugin.video.quasar] github.com/scakemyer/quasar/bittorrent.(*BTPlayer).consumeAlerts(0xc420214480)
23:43:27 T:123145351651328  NOTICE: [plugin.video.quasar] 	/go/src/github.com/scakemyer/quasar/bittorrent/player.go:430 +0x212
23:43:27 T:123145351651328  NOTICE: [plugin.video.quasar] created by github.com/scakemyer/quasar/bittorrent.(*BTPlayer).addTorrent
23:43:27 T:123145351651328  NOTICE: [plugin.video.quasar] 	/go/src/github.com/scakemyer/quasar/bittorrent/player.go:134 +0x7a0
23:43:27 T:123145329496064   ERROR: [plugin.video.quasar] Traceback (most recent call last):
23:43:27 T:123145329496064   ERROR: [plugin.video.quasar]   File "/Users/arkat/Library/Application Support/Kodi/addons/plugin.video.quasar/resources/site-packages/quasar/navigation.py", line 112, in run
```

**Comment by scakemyer** [comment]
Need more info here...

**Comment by scakemyer** [comment]
Was fixed in v0.9.46 by scakemyer/quasar@83e0452
